quasar/main.tf

```diff
@@ -343,6 +343,107 @@ resource "aws_db_parameter_group" "quasar-qa-pg11" {
   }
 }
 
+resource "aws_db_parameter_group" "quasar-qa-pg12" {
+  name   = "quasar-qa-pg12"
+  family = "postgres12"
+
+  # Required for DMS CDC Replication:
+  # https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.PostgreSQL.html#CHAP_Source.PostgreSQL.v10
+  parameter {
+    name         = "rds.logical_replication"
+    value        = "1"
+    apply_method = "pending-reboot"
+  }
+
+  # Required for DMS CDC Replication:
+  # https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.PostgreSQL.html#CHAP_Source.PostgreSQL.v10
+  parameter {
+    name         = "max_logical_replication_workers"
+    value        = "500"
+    apply_method = "pending-reboot"
+  }
+
+  # Recommended by PGTuner tool: https://pgtune.leopard.in.ua/#/
+  # Sets effective RAM available to PG Query planner before using disk.
+  parameter {
+    name  = "effective_cache_size"
+    value = "12000000"
+  }
+
+  # Recommended by PGTuner tool: https://pgtune.leopard.in.ua/#/
+  # Amount of RAM available for cleanup tasks like vacuum, reindex, etc.
+  parameter {
+    name  = "maintenance_work_mem"
+    value = "2000000"
+  }
+
+  # Recommended by PGTuner tool: https://pgtune.leopard.in.ua/#/
+  # Amount of RAM available for joins/sort queries per connection.
+  parameter {
+    name  = "work_mem"
+    value = "26214"
+  }
+
+  # Recommended to only allow SSL connections from clients.
+  parameter {
+    name  = "rds.force_ssl"
+    value = "1"
+  }
+
+  # Only log queries with a duration longer than this to get slow queries.
+  parameter {
+    name  = "log_min_duration_statement"
+    value = "1000"
+  }
+
+  # Only log slow queries.
+  parameter {
+    name  = "log_statement"
+    value = "none"
+  }
+
+  parameter {
+    name  = "log_duration"
+    value = "0"
+  }
+
+  # Testing turning synchronization off to see if improves performance
+  parameter {
+    name  = "synchronous_commit"
+    value = "off"
+  }
```

Comment:
Is this a new thing we're testing? If not, did we see an improvement before? If there's a ticket, let's link it here.

Reply:
This was part of some performance work we tested out about a year ago, when investigating some performance gremlins in the machine. Tried to look for a ticket attached to this, but couldn't find it. We didn't see a measurable performance improvement either way, so happy to remove it. Thoughts?

Reply:
Yeah, let's remove this if we didn't find any improvement.